client/components/admin/AdminTeams/imports/AdminTeamModalGeneral.jsx

```diff
@@ -0,0 +1,56 @@
+import { Meteor } from 'meteor/meteor';
+import React, { Component, } from 'react';
+import { Button, Icon, Grid } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import AdminTeamUserListTracker from './AdminTeamUserList';
+
+class AdminTeamModalGeneral extends Component {
+    _toggleCheckedIn(e) {
+        e.preventDefault();
+        const { team } = this.props;
+        if (!confirm(`Toggle check-in for team ${team.name}?`)) return;
+
+        Meteor.call('team.checkin.toggle', team._id, (err, res) => {
+            if (err) {
+                alert(err);
+            }
+        });
+    }
+    render() {
+        const { team } = this.props;
+
+        return (
+            <div>
+                <Grid columns={3}>
+                    <Grid.Column>
+                        <strong>Team _id:</strong> {team._id}
+                    </Grid.Column>
```

Comment:
Indentation for this file seems to be 4 spaces or tabs, rather than 2 spaces

Reply:
It seems a lot of files do it differently. IMO, so long as it's internally consistent, it doesn't matter to me. Objection if I live it as-is?

Reply:
Okay, fine to leave it then. Thanks!